File: python/conversor_fahrenheit_celsius.py
# Logica de Programação - Excercício 
print("Conversor Fahrenheit para Celsius")
valor_da_temperatura = float(input("Escreva a temperatura em fahrenheit: ")) # Variável que recebe o valor da temperatura em fahrenheit
def converter_fahrenheit_para_celsius(): # Função que converte o valor da temperatura em fahrenheit para celsius
    celsius = (valor_da_temperatura - 32) * 5 / 9
    return print("A temperatura em grau celsius é: {}{}".format(celsius, "°C")) # Retorna o valor da temperatura em celsius
converter_fahrenheit_para_celsius()
    

File: nodejs/conversor_fahrenheit_celsius.js
const readline = require('readline') // Importa o módulo readline
// Cria uma interface de leitura e escrita
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})
// Pergunta ao usuário qual a temperatura em Fahrenheit
rl.question('Qual a temperatura em Fahrenheit: ', (fahrenheit) => {
    // Converte a temperatura para Celsius
    async function converter_fahrenheit_para_celsius(){
        let celsius = await (fahrenheit - 32) * 5 / 9 // Calcula a temperatura
        return console.log(`A temperatura em Celsius é ${celsius}°C`) // Imprime o resultado
    }
    // Chama a função
    converter_fahrenheit_para_celsius()
    // Fecha a interface
    rl.close()
})